datastore/datastore.py

```diff
@@ -349,3 +354,44 @@ def exists(self):
             return elastic_search.create.exists(connection=self._client_or_connection, index_name=self._store_name)
 
         return False
+
+    def external_api_update_entity(self, dictionary_name, dictionary_data, language_script, **kwargs):
```

Comment:
Should be renamed to just `update_entity_data`

Reply:
Done